force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,9 +196,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
+        // Define the correct conversion factor (constant)
+        final Integer MILLISECONDS_PER_MINUTE = 60000;   
```

Comment:
Typically constants get defined at the top of the class, which is why the code scanner is complaining of the naming convention.